geomesa-tools/examples/ingest/example1.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+geomesa ingest -u user -p password -i inst -z zoo -c catalog -conf example1.conf example1.csv
```

Comment:
yeah i think doing all --long-commands would have been better...or may having just added shorts for really really common things like -u, -p etc